docs/topics/native/native-objc-interop.md

```diff
@@ -534,13 +534,44 @@ func foo(block: (KotlinInt) -> KotlinUnit)
 
 And you can call it like this:
 
-```kotlin
+```swift
 foo {
     bar($0 as! Int32)
     return KotlinUnit()
 }
 ```
 
+#### Explicit names in block types
+
+You can add explicit names to Kotlin's block types for exported Objective-C headers. Without them, Xcode's autocompletion
```

Comment:
I guess calling generated ObjC from ObjC in XCode? As said below it doesn't affect calling from Swift, right?

Reply:
Thank you for making it more clear